.github/workflows/release-cli.yml

```diff
@@ -0,0 +1,42 @@
+name: Create Github Release with CLI Jar and Distribution
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+
+jobs:
+  release-job:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.tag-job.outputs.tag }}
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+      - name: Build jar with latest tagged version
+        run: |
+          ./gradlew --build-cache :cli:jar
+          ./gradlew --build-cache :cli:assembleDist
+      - name: Create tagged release with CLI jar
+        run: gh release create ${{ needs.tag-job.outputs.tag }} 'cli/build/libs/java-pfb-cli-${{ needs.tag-job.outputs.tag }}.jar' 'cli/build/distributions/cli-${{ needs.tag-job.outputs.tag }}.zip' --generate-notes --verify-tag
+        env:
+          GH_TOKEN: ${{ github.token }}
+      - name: Notify slack on failure
```

Comment:
Since this is run from inside another workflow, does it need its own slack notification? It's required if it were a dispatched workflow since it run asynchronously. But if it's only run from within another workflow, and that one has slack notifications, could the calling workflow report the error instead?

Reply:
Yeah, that's reasonable, we can just have the single slack notification.